specification/blueprint/resource-manager/Microsoft.Blueprint/preview/2017-11-11-preview/blueprint.json

```diff
@@ -0,0 +1,1703 @@
+          "type": "string",
+          "readOnly": true,
+          "description": "Link to the next page of results."
+        }
+      }
+    },
+    "PublishedBlueprintList": {
+      "type": "object",
+      "title": "BlueprintList",
+      "description": "List of published Blueprints",
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "List of published Blueprints.",
+          "items": {
+            "$ref": "#/definitions/PublishedBlueprint"
+          }
+        },
+        "nextLink": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Link to the next page of results."
+        }
+      }
+    },
+    "ArtifactList": {
+      "type": "object",
+      "description": "List of Blueprint artifacts",
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "List of Blueprint artifacts.",
+          "items": {
+            "$ref": "#/definitions/Artifact"
+          }
+        },
+        "nextLink": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Link to the next page of results."
+        }
+      }
+    },
+    "AssignmentList": {
+      "type": "object",
+      "description": "List of Blueprint assignments",
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "List of Blueprint assignments.",
+          "items": {
+            "$ref": "#/definitions/Assignment"
+          }
+        },
+        "nextLink": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Link to the next page of results."
+        }
+      }
+    },
+    "ResourceProviderOperationList": {
+      "description": "Result of the request to list operations.",
+      "readOnly": true,
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ResourceProviderOperation"
+          },
+          "description": "List of operations supported by this resource provider."
+        }
+      }
+    },
+    "ResourceProviderOperation": {
+      "description": "Supported operation of this resource provider.",
+      "readOnly": true,
+      "properties": {
+        "name": {
+          "description": "Operation name, in format of {provider}/{resource}/{operation}",
+          "type": "string"
+        },
+        "display": {
+          "description": "Display metadata associated with the operation.",
+          "properties": {
+            "provider": {
+              "description": "Resource provider: Microsoft Blueprint.",
+              "type": "string"
+            },
+            "resource": {
+              "description": "Resource on which the operation is performed.",
+              "type": "string"
+            },
+            "operation": {
+              "description": "Type of operation: get, read, delete, etc.",
+              "type": "string"
+            }
```

Comment:
are we missing a `description` property here? this validation error https://travis-ci.org/Azure/azure-rest-api-specs/jobs/427439018#L618 says provided example has description property in the response payload but the model does not have such a property.